packages/mobile/src/identity/securityCode.ts

```diff
@@ -0,0 +1,119 @@
+import {
+  ActionableAttestation,
+  AttestationsWrapper,
+} from '@celo/contractkit/lib/wrappers/Attestations'
+import { PhoneNumberHashDetails } from '@celo/contractkit/src/identity/odis/phone-number-identifier'
+import { GetAttestationRequest } from '@celo/utils/lib/io'
+import { Address } from '@celo/utils/src/address'
+import BigNumber from 'bignumber.js'
+import Logger from 'src/utils/Logger'
+
+const TAG = 'identity/securityCode'
+
+const cache: { [key: string]: string } = {}
+
+function hashAddressToSingleDigit(address: Address): number {
+  return new BigNumber(address.toLowerCase()).modulo(10).toNumber()
+}
+
+export function getSecurityCodePrefix(issuerAddress: Address) {
```

Comment:
Should this code not live in ContractKit or Utils? I.e. other wallets will need to do this too right?

Reply:
Well, if other wallet would be implementing the short code verification, than yes. We can move it to `Utils` I think.

Reply:
I think a lot of the current implementation exists in the Attestation Wrapper (though not saying that's the most appropriate place for it...)